packages/unlock-js/src/locksmithService.ts

```diff
@@ -1,4 +1,4 @@
-import fetch, { Headers } from 'cross-fetch'
+import fetch from 'node-fetch'
```

Comment:
Keep `cross-fetch` import with Headers here since this is a library for both backend and frontend. We will remove this cross-fetch package when upgrading to node v18.